libdnf.spec

```diff
@@ -240,7 +248,7 @@ popd
 
 %find_lang %{name}
 
-%if 0%{?rhel} && 0%{?rhel} <= 7
+%if (0%{?rhel} && 0%{?rhel} <= 7) || 0%{?is_opensuse}
```

Comment:
`0%{?is_opensuse}` -> `0%{?suse_version}`

Reply:
I've changed the macro according to your suggestion.